app/assets/stylesheets/layouts/_header_navbar.scss

```diff
@@ -24,16 +103,20 @@ body {
   &.headroom--pinned {
     top: 0;
   }
+}
 
-  .navbar-inner {
-    background: transparent;
-    border: none;
-  }
-
-  .nav {
-    li {
-      a {
-        color: $bodyBackground;
+@media (max-width: 767px) {
+  .navbar-default .navbar-nav .open .dropdown-menu {
+    > li > a {
+      color: $col-default;
+      &:hover, &:focus {
+        color: $col-highlight;
+      }
+    }
+    > .active {
+      > a, > a:hover, > a:focus {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 6<br>Each selector in a comma sequence should be on its own line